tests/parsing/CliParsingTest.py

```diff
@@ -71,3 +71,8 @@ def test_check_conflicts(self):
         with self.assertRaisesRegex(SystemExit, '2') as cm:
             check_conflicts(sections)
             self.assertEqual(cm.exception.code, 2)
+
+        sections = parse_cli(arg_list=['--format', '--json'])
+        with self.assertRaisesRegex(SystemExit, '2') as cm:
+            check_conflicts(sections)
+            self.assertEqual(cm.exception.code, 2)
```

Comment:
This `self.assertEqual` won't be reached when inside the `with` block. Whereby that's strangely done in all tests above o.O This is confusing. Something must raise a `SystemExit` inside the `with` block. Otherwise the `self.assertRaisesRegex` would fail. And the `self.assertEqual` doesn't raise `SystemExit`. So `check_conflicts` surely raises that, and all `cm.exception.code` checks must be moved out of the `with` blocks.

Correct me if I'm wrong :D 

And please also check the error message :)

**CC** @aptrishu